src/Microsoft.AspNetCore.Mvc.Razor/Compilation/IRoslynCompilationService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.CodeAnalysis.Emit;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Compilation
+{
+    /// <summary>
+    /// A <see cref="ICompilationService"/> that uses Roslyn.
+    /// </summary>
+    public interface IRoslynCompilationService : ICompilationService
```

Comment:
If we're going to start re-working how the CompilationService is talk with @cjqian on what she had to do to tie into the existing compilation service. Would rather ensure we have a coherent set of changes that enable VC TagHelpers and this.